external-crates/move/crates/move-vm-runtime/src/native_functions.rs

```diff
@@ -158,6 +159,46 @@ impl<'b> NativeContext<'_, 'b> {
         }
     }
 
+    // TODO: This is a bit hacky right now since we need to pass the store, however this is only
+    // used in test scenarios so we have some special knowledge that makes this work. In the new VM
+    // however this is _MUCH_ nicer as we don't need to pass the datastore as the VM's linkage
+    // tables must have the type present.
+    pub fn type_tag_to_fully_annotated_layout(
```

Comment:
maybe give it a strange name like `type_tag_to_fully_annotated_layout_for_test_scenario` (not this stops people as we saw with the vector construction)